tests/components/ozw/test_light.py

```diff
@@ -149,3 +149,335 @@ async def test_light(hass, light_data, light_msg, sent_messages):
+    await hass.async_block_till_done()
+
+    state = hass.states.get("light.led_bulb_6_multi_colour_level")
+    assert state is not None
+    assert state.state == "on"
+    assert state.attributes["white_value"] == 190
+
+
+async def test_no_cw_light(
+    hass, light_no_cw_data, light_msg, light_rgb_msg, sent_messages
+):
+    """Test setting up config entry."""
+    receive_message = await setup_ozw(hass, fixture=light_no_cw_data)
+
+    # Test loaded no cw support
+    state = hass.states.get("light.led_bulb_6_multi_colour_level")
+    assert state is not None
+    assert state.state == "off"
+
+    # Turn on the light
+    white_color = 190
+    await hass.services.async_call(
+        "light",
+        "turn_on",
+        {
+            "entity_id": "light.led_bulb_6_multi_colour_level",
+            "white_value": white_color,
+        },
+        blocking=True,
+    )
+    assert len(sent_messages) == 2
+    msg = sent_messages[-2]
+    assert msg["topic"] == "OpenZWave/1/command/setvalue/"
+    assert msg["payload"] == {"Value": "#000000be00", "ValueIDKey": 659341335}
+
+    # Feedback on state
+    light_msg.decode()
+    light_msg.payload["Value"] = byte_to_zwave_brightness(255)
+    light_msg.encode()
+    light_rgb_msg.decode()
+    light_rgb_msg.payload["Value"] = "#000000be00"
+    light_rgb_msg.encode()
+    receive_message(light_msg)
+    receive_message(light_rgb_msg)
+    await hass.async_block_till_done()
+
+    state = hass.states.get("light.led_bulb_6_multi_colour_level")
+    assert state is not None
+    assert state.state == "on"
+    assert state.attributes["white_value"] == 190
+
+
+async def test_wc_light(hass, light_wc_data, light_msg, light_rgb_msg, sent_messages):
+    """Test setting up config entry."""
+    receive_message = await setup_ozw(hass, fixture=light_wc_data)
+
+    # Test loaded only white LED support
+    state = hass.states.get("light.led_bulb_6_multi_colour_level")
+    assert state is not None
+    assert state.state == "off"
+
+    # Turn on the light
+    new_color = 190
+    await hass.services.async_call(
+        "light",
+        "turn_on",
+        {"entity_id": "light.led_bulb_6_multi_colour_level", "color_temp": new_color},
+        blocking=True,
+    )
+    assert len(sent_messages) == 2
+    msg = sent_messages[-2]
+    assert msg["topic"] == "OpenZWave/1/command/setvalue/"
+    assert msg["payload"] == {"Value": "#0000001be4", "ValueIDKey": 659341335}
+
+    # Feedback on state
+    light_msg.decode()
+    light_msg.payload["Value"] = byte_to_zwave_brightness(255)
+    light_msg.encode()
+    light_rgb_msg.decode()
+    light_rgb_msg.payload["Value"] = "#0000001be4"
+    light_rgb_msg.encode()
+    receive_message(light_msg)
+    receive_message(light_rgb_msg)
+    await hass.async_block_till_done()
+
+    state = hass.states.get("light.led_bulb_6_multi_colour_level")
+    assert state is not None
+    assert state.state == "on"
+    assert state.attributes["color_temp"] == 191
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5844602)